assets/app/view/game/round/bidder.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'view/game/actionable'
+require 'view/game/bank'
+require 'view/game/buy_sell_shares'
+require 'view/game/company'
+require 'view/game/corporation'
+require 'view/game/par'
+require 'view/game/players'
+require 'view/game/sell_shares'
+require 'view/game/stock_market'
+require 'view/game/undo_and_pass'
+
+module View
+  module Game
+    module Round
+      module Bidder
+        def render_bid(entity, corporation)
```

Comment:
i think this would be better as a widget with needs

Reply:
done